packages/pn-personafisica-webapp/src/components/Contacts/DigitalContactsCodeVerification.context.tsx

```diff
@@ -388,33 +382,35 @@ const DigitalContactsCodeVerificationProvider: FC<{ children?: ReactNode }> = ({
         <DialogTitle id="dialog-title">
           {t(`common.duplicate-contact-title`, { value: modalProps.value, ns: 'recapiti' })}
         </DialogTitle>
-        <DialogContent>
+        <PnDialogContent>
           <DialogContentText id="dialog-description">
             {t(`common.duplicate-contact-descr`, { value: modalProps.value, ns: 'recapiti' })}
           </DialogContentText>
-        </DialogContent>
-        <DialogActions>
+        </PnDialogContent>
+        <PnDialogActions>
           <Button onClick={handleDiscard} variant="outlined">
             {t('button.annulla')}
           </Button>
           <Button onClick={handleConfirm} variant="contained">
             {t('button.conferma')}
           </Button>
-        </DialogActions>
-      </Dialog>
-      <Dialog open={pecValidationOpen} data-testid="validationDialog">
-        <DialogTitle id="dialog-title" sx={{ pt: 4, px: 4 }}>
+        </PnDialogActions>
+      </PnDialog>
+      <PnDialog open={pecValidationOpen} data-testid="validationDialog">
```

Comment:
Checks the comments left on CodeModal about distance between Title and Content, and Content and Actions